pop/POPAnimator.mm

```diff
@@ -347,8 +366,25 @@ - (id)init
   _displayLink.paused = YES;
   [_displayLink addToRunLoop:[NSRunLoop mainRunLoop] forMode:NSRunLoopCommonModes];
 #else
-  CVDisplayLinkCreateWithActiveCGDisplays(&_displayLink);
-  CVDisplayLinkSetOutputCallback(_displayLink, displayLinkCallback, (__bridge void *)self);
+  CVReturn ret = CVDisplayLinkCreateWithActiveCGDisplays(&_displayLink);
+  if (kCVReturnSuccess != ret) {
```

Comment:
IMO, if you write left to right it is more natural to have receiver on left side.